src/Compilers/Core/CodeAnalysisTest/Text/EncodedStringTextTests.cs

```diff
@@ -97,7 +97,7 @@ public void CheckSum_SHA256()
         public void Decode_NonUtf8()
         {
             var utf8 = new UTF8Encoding(false, true);
-            var encoding = Encoding.GetEncoding(name: "Latin1");
+            var encoding = Encoding.Default;
```

Comment:
@AlekseyTs Unfortunately, as it stands right now, CodePage1252Encoding is a decoder only, so it won't work here.